rides/__init__.py

```diff
@@ -1,17 +1,24 @@
 ####################################
 # File name: __init__.py           #
 # Author: Ayush Goel & Fred Rybin  #
+# Contributors: Joe Abbate         #
 ####################################
 from subprocess import check_output
 import datetime
 import os
+from typing import TypedDict
+from flask_mail import Mail, Message
 import pytz
 from flask_pyoidc.flask_pyoidc import OIDCAuthentication
 from flask_pyoidc.provider_configuration import ProviderConfiguration, ClientMetadata
-from flask import Flask, render_template, send_from_directory, redirect, url_for, g
+from flask import Flask, render_template, send_from_directory, redirect, url_for, g, request, session
 from flask_sqlalchemy import SQLAlchemy
 from flask_wtf.csrf import CSRFProtect
 from flask_login import login_user, logout_user, login_required, LoginManager, current_user
+import pymysql
+import requests
```

Comment:
Similar for `requests`

Reply:
Changed in commit https://github.com/ComputerScienceHouse/csh-rideboard/pull/22/commits/711d1a4ad688e98faa8a7ae76c4c812854517b00